core/widgets/SubmitButtonWidget.php

```diff
@@ -42,9 +42,31 @@ class SubmitButtonWidget extends Widget
     public $options = [];
 
     /**
+     * @since 1.0.24 First time this was introduced
```

Comment:
Just @since 1.0.24 is enough. thanks perfect. Please add since after @var (https://github.com/luyadev/luya/blob/master/docs/guide/luya-guideline.md#phpdoc)

```php
/**
 * @var ...
 * @since
 */
```